tests/05_backend/test_backend_slurm.py

```diff
@@ -184,13 +184,13 @@ def wait_and_return():
             fut = executor.submit(wait_and_return)
             debug(fut)
 
-            # Wait until the SLURM job goes from PENDING to RUNNING
+            # Wait until the SLURM job goes from PENDING to SUBMITTED
```

Comment:
Note that in this module RUNNING corresponds to a SLURM status, and it should not be modified.